docs/man/docker-build.1.md

```diff
@@ -30,6 +31,9 @@ When a Git repository is set as the **URL**, the repository is used
 as context.
 
 # OPTIONS
+**-f**, **--file**=*Dockerfile*
+   Location of the Dockerfile to use. The file must be within the build context. The default is *Dockerfile*.
```

Comment:
ok I updated the docs here to make it clear that its relative to the current dir. See if it helps.